assets/wizards/readerRevenue/index.js

```diff
@@ -195,6 +195,7 @@ class ReaderRevenueWizard extends Component {
 				return [];
 			}
 			return [
+				platformField,
```

Comment:
Why should platform settings be the first tab? This is most likely the least used tab, since a site would set the platform once and then visit the Reader Revenue wizard to tweak the settings, e.g. Donations tab.

Reply:
By default, when you visit the Reader Revenue, you see the Platform screen first, I didn't change (I guess it's because the path is set to `\`... I'm not quite sure why, Jeff did this). So the only thing I changed in the PR is reorganising the tabs so that Platform is also 1st in the `TabbedNavigation`.

Having said that, I agree, Platform should be the last tab (`/platform`?) and see the Donations screen first but I'm unsure on how to do this.

Reply:
Ah, ok. Done in 3193c40b2c402e5f8f010edfa72289fd4b40e73e. I've removed the has-platform-set check since there is a default (the Newspack platform).

Reply:
Moved Platform to the last menu item: https://github.com/Automattic/newspack-plugin/pull/1114/commits/b0bc43e8df809e3f0bb13dbf620e2906df587d07